hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveSessionTest.java

```diff
@@ -700,6 +706,18 @@ public void testBatching(TestContext context) {
 		);
 	}
 
+	@Test
+	public void testBatchingConnection() {
+		Stage.Session session = getSessionFactory().openSession();
```

Comment:
Sorry, one thing I've just noticed... You are not using the method `openSession` in the superclass, meaning that the session doesn't get closed at the end of the test.